docs/sce_file_contents.md

```diff
@@ -130,12 +132,11 @@ metadata(sce) # experiment metadata
 | `cluster_algorithm` | The algorithm used to perform graph-based clustering of cells. Only present for `processed` objects |
 | `cluster_weighting` | The weighting approach used during graph-based clustering. Only present for `processed` objects |
 | `cluster_nn`        | The nearest neighbor parameter value used for the graph-based clustering. Only present for `processed` objects |
-| `celltype_methods` | If cell type annotation was performed, a vector of the methods used for annotation. May include `"singler"` and/or `"cellassign"`. Only present for `processed` objects |
+| `celltype_methods` | If cell type annotation was performed, a vector of the methods used for annotation. May include `"submitter"`, `"singler"` and/or `"cellassign"`. If submitter cell-type annotations are available, this will be present in all objects, otherwise this item is only present for `processed` objects |
 | `singler_results` | If cell typing with `SingleR` was performed, the full result object returned by `SingleR` annotation. Only present for `processed` objects |
-| `singler_reference_name` | If cell typing with `SingleR` was performed, the name of the [`celldex`](http://bioconductor.org/packages/release/data/experiment/html/celldex.html) reference dataset used for annotation. Only present for `processed` objects |
-| `singler_reference_label` | If cell typing with `SingleR` was performed, the name of the label in the reference dataset used for annotation. Only present for `processed` objects |
```

Comment:
I actually wonder if we should revive this and get it into the SCE metadata. It seems like useful information to have about celltyping, but also something to circle back to if we re-add it in. 
Let me know what you think about this!

----

If we do that, here's how we might go about it:

1. Modify [`classify_SingleR.R`](https://github.com/AlexsLemonade/scpca-nf/blob/development/bin/classify_SingleR.R) to also add the label name into the singler result metadata
2. Add lines here to add info to SCE metadata:
https://github.com/AlexsLemonade/scpca-nf/blob/934df294bbfe2a01d8ffdb94ba9f5d6b432a547d/bin/add_celltypes_to_sce.R#L100-L102
3. Bonus change: We could also change this line to check the label name directly instead of name presence, but either way should be fine. https://github.com/AlexsLemonade/scpca-nf/blob/934df294bbfe2a01d8ffdb94ba9f5d6b432a547d/bin/add_celltypes_to_sce.R#L75

Reply:
I'm good with that. Was there a reason that we removed it in the first place?

Reply:
> Was there a reason that we removed it in the first place?

No, I think it just got lost in the shuffle, especially when we changed the code to check for the ontology df in the model object instead of passing in the label parameter. Thanks for filing the PR for it (expect a review after YI, probably).